### ISSUE TITLE
How do i get RSS feeds from Mangadex?

### ISSUE THREAD

**Issue by hercyle** [Owyn/CSS2RSS#3]
from any specified title, like
[Mushoku Tensei](https://mangadex.org/title/bd6d0982-0091-4945-ad70-c028ed3c0917/mushoku-tensei-isekai-ittara-honki-dasu)

the tool should only get chapter in `english` ignoring other languages.
in rss title it could be: `{title} - {chapter}`
in rss description it could be: `{scans group} - {date}`

my bad for asking, but i've got no idea on how to build a parser for this site.

**Comment by Owyn** [comment]
Well, I tried too but gave up, cuz mangadex requires both javascript and authorization to make it work and I use the nowebengine rssguard version for less resource usage

you can filter languages in your account settings then log-in, then look for `.chapter-link` ?

or... you could just join the TL-group's discord and get notifications there?

**Comment by hercyle** [comment]
that's a pity.
i tried the `.chapter-link` thing but no luck.

i'll close the issue. as Mangadex makes it difficult for this tool to get rss from.

**Comment by Crehet** [comment]
rssguard now seem to have done something cool to now be able to get feeds from JS populated sites. have you tested if it's still not possible?

**Comment by Owyn** [comment]
I haven't tested it since I'm using light version of rssguard without that,

but if it's now possible - then it's possible I guess?

**Comment by Crehet** [comment]
im also using the lite version. when editing a feed -- in `Source` there's the option to select the `built-in browser with JS support`

**Comment by Owyn** [comment]
so - try it and see if it gives different results?

**Comment by Crehet** [comment]
im like the OP. i dunno how to get feeds from MD as well. i've tried the OP's manga link and then `python css2rss.py ".chapter-link"` but no luck. also, one really nees the webengine version to use that source option, and i tested getting the feed with the webengine version.

**Comment by Owyn** [comment]
What are you expecting in .chapter-link? it's just text there and you need a link
![image](https://github.com/Owyn/CSS2RSS/assets/1309656/33667812-f836-4c58-8dc6-348632b11bc4)

**Comment by Owyn** [comment]
Try  `python css2rss.py "a.items-center" "!Your manga title"`

**Comment by Owyn** [comment]
@DawEdhel that might be due to you using fancy pseudo-selectors such as `:scope` and `:has`, I doubt BS4 module used here even supports them...

**Comment by Crehet** [comment]
> @DawEdhel that might be due to you using fancy pseudo-selectors such as `:scope` and `:has`, I doubt BS4 module used here even supports them...

no. @DawEdhel is quite right. rssguards hangs even with very simple use of css2rss while trying `built-in browser with JS support`.

so, unless rssguard fixes that hanging, i've got no way to see a result from the command suggestions you guys provided.

**Comment by Owyn** [comment]
We could try asking @martinrotter if it works for him?

with
example url: `https://mangadex.org/title/bd6d0982-0091-4945-ad70-c028ed3c0917/mushoku-tensei-isekai-ittara-honki-dasu`
scrapper: `python css2rss.py "a.items-center" "!Your manga title"`

and that new option `built-in browser with JS support` turned on

**Comment by Crehet** [comment]
>     3. I've even tried your `python css2rss.py "a.items-center" "!Your manga title"` and I was getting the same hanging results. Something unique happens within this website, as I had already used successfully this built-in feature with another sites.

can you provide an example link to test out  `built-in browser with JS support`?


> We could try asking @martinrotter if it works for him or if we're doing something we aren't supposed to?
> 
> with example url: `https://mangadex.org/title/bd6d0982-0091-4945-ad70-c028ed3c0917/mushoku-tensei-isekai-ittara-honki-dasu` scrapper: `python css2rss.py "a.items-center" "!Your manga title"`
> 
> and that new option `built-in browser with JS support` turned on

yeah. however, my brain is not  up for making constructive issues. while i could build one, i'd prefer to leave it to someone with more knowledge about it. if i see no issue that highlights this particular issue, i'd probaby open one myelf, though.
